### ISSUE TITLE
FutureWarning: Assigning CRS to a GeoDataFrame without a geometry

### ISSUE THREAD

**Issue by martinfleis** [jsignell/dask-geopandas#21]
Running dask-geopandas with geopandas master raises the following warning: `/opt/conda/lib/python3.7/site-packages/dask_geopandas/backends.py:54: FutureWarning: Assigning CRS to a GeoDataFrame without a geometry column is now deprecated and will not be supported in the future.`

It does that on every step.

```python
ddf = dask_geopandas.from_geopandas(df, npartitions=14)
ddf['area'] = ddf.buildings.area
```

In GeoPandas, we have moved CRS attribute to GeometryArray level. Since we do not really want to have GeoDataFrame without a geometry but with CRS, we aim to deprecate that option. I guess than in here it is better to also store the copy on dask.GeoDataFrame level, but we have to figure out if it is better to keep the existing support in geopandas or change the implementation here.

**Comment by jorisvandenbossche** [comment]
Do you have a reproducible example? Or can you show the stacktrace if you turn the warning into an error?

I don't see anything with this simple example (using master of all):

```
import geopandas
import dask.dataframe as dd

df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres")).to_crs("EPSG:3857")
ddf = dask_geopandas.from_geopandas(df, npartitions=4)

ddf['area'] = ddf.geometry.area

```

**Comment by martinfleis** [comment]
I missed one important line, sorry. It happens with renamed geometry column.

```python
import geopandas
import dask_geopandas

df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres")).to_crs("EPSG:3857")
df = df.rename_geometry('geom')
ddf = dask_geopandas.from_geopandas(df, npartitions=4)

ddf['area'] = ddf.geometry.area
```

Thinking about it now, we may have to fix that in GeoPandas, because you can reproduce it without dask as 

```python
df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
df = df.rename_geometry('geom')
geopandas.GeoDataFrame(df, crs=4326)
```

With geometry column `geometry` this is either silent if you pass the same CRS or warns about CRS mismatch if you pass different one. With a different name, it does not see existing geometry in the first place. Even `geopandas.GeoDataFrame(df).geometry` breaks if you rename the column before.

**Comment by jorisvandenbossche** [comment]
Ah, yes, that seems indeed something to solve in geopandas (or at least the pure geopandas snippet should also not warn I think?)

**Comment by martinfleis** [comment]
Yeah, in the creation of GeoDataFrame from another GeoDataFrame we should capture that and pass metadata (geometry column, maybe crs for now) to a new one. I'll move this issue to geopandas repo.
